### ISSUE TITLE
1.9.0 Issues

### ISSUE THREAD

**Issue by R4Z0R49** [DayZMod/DayZ#926]
Any issues with 1.9.0 please post here

**Comment by skiperh1n1** [comment]
Hello, i seen one bug(or not) when i want kill other player hes respawning on same position with loot, what happend?

**Comment by skigoggles** [comment]
are you sure you killed him or did he combat log?

**Comment by ebayShopper** [comment]
@skiperh1n1 the server has most likely not updated their dayz_server.pbo to the 1.9 version. That will cause server_playerDied.sqf to error out. Make sure the server pbo, mission and addons are updated to 1.9 release here: https://forums.dayz.com/topic/237744-dayzmod-19-release/

**Comment by skiperh1n1** [comment]
@ebayShopper Yep, its a 1.9.0 Id install server clean, files take from this github repository. Server_playerDied.sqf dont have any errors.

**Comment by oiad** [comment]
@skiperh1n1 is the server running infiSTAR?

**Comment by skiperh1n1** [comment]
@oiad no, hes dont, just bec

**Comment by skiperh1n1** [comment]
One more [Skipping objectID 391 load because of invalid data in db]

**Comment by skigoggles** [comment]
different issue, but please post the contents of the row in object_data with objectid=391. I've heard a couple of other get this. Both were a row for an Ikarus, but I think that was just a coincidence.

**Comment by skiperh1n1** [comment]
Oh, id slove problem with death, now its work excelent
  And about ObjectID 391, its a problem with Volha_2_TK_CIV_EP1 but its be in game and work. Magic...

**Comment by skiperh1n1** [comment]
Thanks all for help 👍

**Comment by unhappytroll** [comment]
There is **Volha_1_TK_CIV_EP1** and **Volha_2_TK_CIV_EP1** in _vehicle_spawns_ table. But there is no **Volha_2_TK_CIV_EP1** in object_classes, because there is **VolhaLimo_TK_CIV_EP1** class instead.
Which table is correct?

**Comment by skigoggles** [comment]
vehicle_spawns is correct.  object_classes has not been used for many releases so it has not been updated and really should be removed from the repository.

**Comment by unhappytroll** [comment]
Ah, ok. I do not see _object_classes_ in current master branch, but I have 1.9.0 release archives and it is still there. 
And in  current version looks like all three classes is present in _vehicle_spawns_.

**Comment by endreu1** [comment]
Hi, got small problem with joining, some time players get kick with
`Value Restriction #1 "remExField" = [B 1-1-A:1 (nick) REMOTE,any,"JIPrequest"]`
tried to search about it, but i could. recompl mission file, all checked twice. still no luck
where should i start?
thx

**Comment by ebayShopper** [comment]
@endreu1 make sure the server has the latest BE filters from here if it is vanilla:
https://github.com/DayZMod/Battleye-Filters

Your kick is most likely for the brackets `[ ]` which we do not kick for anymore in the newest publicvariableval.txt. If it is not a vanilla server then you need to adjust BE filters yourself to fix any false positives.

**Comment by endreu1** [comment]
It's full vanilla since 2012 year(yes all char data is still there) tried again to update BE filters, all the same
update: tried on 2-nd server with same server config and data base. all is the same ;(
update #2: server restart does not help

**Comment by Havocfive** [comment]
Military Prison- on the ladder that goes to the roof, you still climb up sideways. 
Soda machines- lit up
ATC- light on top that flashes
LOD on new enterables- From a distance some of the new enterable buildings look like they have lights on, but when you zoom in it disappears. The small military building (end of NWAF runway, north of Zino hospital) look like the old tan version from far out but then change to grey when closer/ zoomed in.

**Comment by Jbird206** [comment]
Dismantling object, e.g. sandbag wall or hedgehog. Upon using action menu to disassemble, sometimes errors occur. The sound will play but animation doesn’t happen. Object does not get disassembled. Switching to primary, secondary or melee sometimes resolves issue and normal disassembly can occur.
